driver/src/main/java/org/neo4j/driver/internal/retry/ExponentialBackoffRetryLogic.java

```diff
@@ -45,14 +45,14 @@ public class ExponentialBackoffRetryLogic implements RetryLogic {
     public static final long DEFAULT_MAX_RETRY_TIME_MS = SECONDS.toMillis(30);
 
     private static final long INITIAL_RETRY_DELAY_MS = SECONDS.toMillis(1);
-    private static final double RETRY_DELAY_MULTIPLIER = 1;
+    private static final double RETRY_DELAY_MULTIPLIER = 2.0;
```

Comment:
I'll comment here as it is not possible to comment on a non-changed line. 

Line 328 should probably check that multiplier that is supplied is >= 2.0 rather than 1.0.

Reply:
I have updated the check. There is also a new test that verifies that the default value is `2.0`.